src/components/PlioQuestion.vue

```diff
@@ -269,6 +295,11 @@ i.fas{
   cursor: pointer;
 }
 
+.disabled-div {
+  pointer-events: none;
+  opacity: 0.4;
+}
+
```

Comment:
Added to the class list of skip/revise/submit. We want these buttons to be disabled when loading spinner is showing up